test/helpers/mocks.lua

```diff
@@ -0,0 +1,55 @@
+local M = {}
```

Comment:
Please, add a small description of a purpose of the role.

Reply:
Added.